packages/zowe-explorer/src/webviews/src/utils.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import { useEffect } from "preact/hooks";
+
+export function getVsCodeTheme(): string | null {
+    return document.body.getAttribute("data-vscode-theme-kind");
+}
+
+export const useMutableObserver = (target: Node, callback: MutationCallback, options: MutationObserverInit | undefined = undefined): void => {
```

Comment:
is it ok to remove the default value for options here?
I believe that it will be `undefined` if not provided 😋

Reply:
Resolved in f8c068de